guides-and-tutorials/running-a-signer/signer-quickstart.md

```diff
@@ -0,0 +1,435 @@
+
+Assuming a `Linux x64 glibc` machine, the commands to download and uncompress the node binary look like this:
+
+```bash
+# Enter the node directory
+cd ~/stacks-node
+
+# Download the node binary zip
+wget https://github.com/stacks-network/stacks-core/releases/download/3.0.0.0.0/linux-glibc-x64.zip
+
+# Unzip the node binary archive
+unzip linux-glibc-x64.zip
+```
+
+### Create the configuration file
+
+Create the configuration file required to start the node (be sure to replace `<your_token>` with your auth token value):
+
+{% tabs %}
+
+{% tab title="Mainnet" %}
+
+{% hint style="warning" %}
+
+For mainnet, we strongly recommended that you run your own bitcoin node (you can follow [this](https://github.com/stacksfoundation/miner-docs/blob/main/bitcoin.md) guide) in order to ensure you have no connection issues when downloading bitcoin blocks. A hosted bitcoin node may cause your stacks node to fall behind tip and remain unsynced.
+
+If you run your own bitcoin node, you'll have to update `peer_host` and optionally add `rpc_port`, `peer_port`, `username` and `password` fields under the `[burnchain]` section of the node's configuration file.
+
+{% endhint %}
+
+```bash
+# Set environment variables
+AUTH_TOKEN=<your_token> # Used for signer-node authentication, same token as the one set up in the signer configuration
+
+# Create the node's configuration file
+cat <<EOF> ~/stacks-node/node-config.toml
+[node]
+working_dir = "$HOME/stacks-node/data"
+rpc_bind = "0.0.0.0:20443"
+p2p_bind = "0.0.0.0:20444"
+prometheus_bind = "127.0.0.1:9153" 
+bootstrap_node = "02196f005965cebe6ddc3901b7b1cc1aa7a88f305bb8c5893456b8f9a605923893@seed.mainnet.hiro.so:20444,02539449ad94e6e6392d8c1deb2b4e61f80ae2a18964349bc14336d8b903c46a8c@cet.stacksnodes.org:20444,02ececc8ce79b8adf813f13a0255f8ae58d4357309ba0cedd523d9f1a306fcfb79@sgt.stacksnodes.org:20444,0303144ba518fe7a0fb56a8a7d488f950307a4330f146e1e1458fc63fb33defe96@est.stacksnodes.org:20444"
+stacker = true
+
+[burnchain]
+chain = "bitcoin"
+mode = "mainnet"
+peer_host = "bitcoin.mainnet.stacks.org"
+
+[connection_options]
+auth_token = "$AUTH_TOKEN"
+
+[[events_observer]]
+endpoint = "127.0.0.1:30000"
+events_keys = ["stackerdb", "block_proposal", "burn_blocks"]
+EOF
+```
+
+{% endtab %}
+
+{% tab title="Testnet" %}
+
+```bash
+# Set environment variables
+AUTH_TOKEN=<your_token> # Used for signer-node authentication, same token as the one set up in the signer configuration
+
+# Create the node's configuration file
+cat <<EOF> ~/stacks-node/node-config.toml
+[node]
+working_dir = "$HOME/stacks-node/data"
+rpc_bind = "0.0.0.0:20443"
+p2p_bind = "0.0.0.0:20444"
```

Comment:
not sure if we want this to be open on all interfaces. i think we should default to private instances, and leave it up to the user (i.e. advanced users)if they want to expose the ports outside of localhost. 
```suggestion
rpc_bind = "127.0.0.1:20443"
p2p_bind = "0.0.0.0:20444"
```

not sure if we want this to be open on all interfaces. i think we should default to private instances, and leave it up to the user (i.e. advanced users)if they want to expose the ports outside of localhost. 

at the very least, for the signer we shojldn't allow external RPC requests